pandas/core/generic.py

```diff
@@ -6651,6 +6658,21 @@ def copy(self, deep: bool_t | None = True) -> Self:
         0    [10, 2]
         1     [3, 4]
         dtype: object
+
+        ** Copy-on-Write is set to true: **
+
+        >>> pd.set_option("mode.copy_on_write", True)
```

Comment:
This will set the option to True for all following tests as well, this has to be wrapped in with(pd.options_context...)